src/cache_cf.cc

```diff
@@ -1542,9 +1544,12 @@ static void
 dump_acl_nfmark(StoreEntry * entry, const char *name, acl_nfmark * head)
 {
     for (acl_nfmark *l = head; l; l = l->next) {
-        if (l->nfmark > 0)
+        if (l->nfmark > 0) {
             storeAppendPrintf(entry, "%s 0x%02X", name, l->nfmark);
-        else
+
+            if (l->nfmask != 0xffffffff)
+                storeAppendPrintf(entry, "/%02X", l->nfmask);
```

Comment:
Similarly to parsing discussed in another change request, we ought to reuse the printing code/approach used in Acl::ConnMark::dump() instead of half-duplicating that code and introducing inconsistencies. AFAICT, that reuse would be easy once the ConnMarkConfig class (described in that other change request) is introduced. The existing `std::ostream` printing method (and possibly more) would become available for dumping clientside_mark values.

Reply:
Also, be aware that CONNMARK are 32-bit values not 8-bit so the %02X is plain wrong anyway.